versions.md

```diff
@@ -1,6 +1,7 @@
 Forthcoming
 -----------
 
+* Compilation fix of test `DDSSQLFilterValueTests.test_filtered_value` that does not exist if compiled with `-DGTEST_INDIVIDUALS=OFF`.
```

Comment:
Fixes are not included in the versions.md. The corresponding commit can be removed (and the conflicts will be solved too).